src/NHibernate/Util/DbTypeExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Data;
+
+namespace NHibernate.Util
+{
+	public static class DbTypeExtensions
```

Comment:
As this is a regression fix, I think that we should not expose new type/methods.

Reply:
A fair point. Would you agree with just making internal?

Reply:
I made it internal.

Reply:
Thanks!